decoders/connector/lansitec/nb-iot-badge-tracker/v1.0.0/payload.js

```diff
@@ -0,0 +1,393 @@
+// nb-iot badge tracker
```

Comment:
New decoders that are added must be made in TypeScript and must have a unit test.